include/z64player.h

```diff
@@ -1388,7 +1388,7 @@ void func_80123E90(struct PlayState* play, Actor* actor);
 s32 func_80123F2C(struct PlayState* play, s32 ammo);
 bool Player_IsBurningStickInRange(struct PlayState* play, Vec3f* pos, f32 xzRange, f32 yRange);
 u8 Player_GetStrength(void);
-u8 Player_GetMask(struct PlayState* play);
+s32 Player_GetMask(struct PlayState* play);
```

Comment:
ohhh, surprising. How did we not figure this out earlier?

Since this matches as an `s32`, would it match using the enum as the return type too?
```suggestion
PlayerMask Player_GetMask(struct PlayState* play);
```